packages/metrics_center/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   http: ">=0.13.0 <2.0.0"
 
 dev_dependencies:
+  _discoveryapis_commons: ^1.0.0
```

Comment:
Seems a complain on this dependency in the CQ: https://github.com/flutter/packages/pull/4323/checks?check_run_id=14605635593